### PR TITLE
Refactor dom bindings project

### PR DESCRIPTION
**Pull request by diegomura**
Same project that @langovoi fixed on #170 , but without all the boilerplate of the CRA ejected project. I think that as an example, is more simple and self-explanatory. Thoughts @langovoi ? I don't want you to feel that I rewrite what you did. All I did was a refactoring

### CODE REVIEW

**Comment by langovoi** [comment]
If we want to make `dom-bindings` self-explanatory, we need to remove it from `lerna` and some fixes: https://github.com/langovoi/react-pdf/commit/c2696c30420970c9d1b896b69ba874ef3289cdb0

Also I see you don't correctly set react/react-dom as dependencies, I fixed this: https://github.com/langovoi/react-pdf/commit/c60eef651f6fcc71971c7c1667d94bcb178c36f9

**Comment by diegomura** [comment]
Great @langovoi ! Apparently I missed some things in the hurry yesterday haha.
Can you commit your first changes into this PR and create a new one fixing the react/react-dom dependencies?

**Comment by diegomura** [comment]
I'm sick of these `output.pdf` conflicts. I'll have to see if I find a way of getting rid of them and yet generating the examples on each commit btw

**Comment by langovoi** [comment]
@diegomura I have't access to push to this PR.

I prepared branch, you can merge it to your: https://github.com/langovoi/react-pdf/tree/diegomura/refactor-dom-bindings

**Comment by langovoi** [comment]
After merging of this PR, I will prepare another one with fix of dependencies for packages.

File: examples/dom-bindings/.babelrc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  presets: [
+    "es2015",
```
[the remark this answers is not in the document]
Comment (langovoi) [reply examples/dom-bindings/.babelrc]: I think it is better, but does not matter for dom-bindings example.

File: packages/react-pdf/package.json [hunk 2]
```diff
@@ -18,6 +18,8 @@
   "license": "MIT",
   "devDependencies": {
     "babel-cli": "^6.24.0",
+    "react": "16.0.0-alpha.4",
```
[the remark this answers is not in the document]
Comment (langovoi) [reply packages/react-pdf/package.json]: I think it is my changes, which was done before reconciler updated.

Think we can remove it from this PR, I will provide another PR for fix `package.json` and `yarn.lock` in `packages/*`

**Pull request merged**